addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -33,25 +62,35 @@
             <field name="arch" type="xml">
                 <search string="Search report">
                     <field name="channel_name"/>
-                    <field name="partner_id"/>
-                    <filter name="last_24h" string="Last 24h" domain="[('start_date','&gt;', (context_today() - datetime.timedelta(days=1)).strftime('%Y-%m-%d') )]"/>
-                    <filter name="start_date_filter" string="This Week" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(weeks=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
-                    <filter name="last_30_days" string="Last 30 Days" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(months=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
-                    <filter name="this_year" string="This Year" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(years=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
+                    <field name="partner_id" string="Agent"/>
+                    <field name="channel_name" string="Channel"/>
+                    <field name="country_id" string="Country"/>
+                    <field name="chatbot_partner_id" string="Chatbot"/>
+                    <field name="visitor_partner_id" string="Customer"/>
+                    <field name="operator_expertise_id" string="Expertise"/>
+                    <filter name="my_session" domain="[('partner_id.user_id', '=', uid)]" string="My Sessions"/>
+                    <separator/>
+                    <filter name="escalted_session" string="Escalated" domain="[('is_without_answer', '=', 0)]"/>
+                    <filter name="not_answered" string="Not Answered" domain="[('is_without_answer', '=', 1)]"/>
+                    <filter name="none_available" string="No one Available" domain="[('partner_id', '=' , False)]"/>
```

Comment:
I don't think this domain accurately reflects the case where no one was available, for several reasons:

- Chats aren't created when no one is available. The `partner_id` relies on `discuss_channel.livechat_operator_id`, so if it's empty, it can only be because the partner was deleted.
- Even if we were to look for the presence of an agent in the history, there's nothing that guarantees the bot actually reached a `"forward_operator"` step.